snmp/postfixdetailed

```diff
@@ -500,10 +507,10 @@ my $data=$received."\n".
 	$rardnf."\n".
 	$rarnfqa."\n".
 	$iuscp."\n".
+	$msefl."\n".
```

Comment:
Why is this moved further up? Doesn't it mean this will now be broken for existing users?